Modules/Bridge/NumPy/wrapping/itkPyVectorContainer.wrap

```diff
@@ -1,19 +1,33 @@
 itk_wrap_class("itk::PyVectorContainer")
+  UNIQUE(identifiertypes "${WRAP_ITK_SCALAR};UC;UI;UL;ULL")
```

Comment:
I don't know but will ask anyway ... would using `${ITKM_IT}` instead of `UL` be more appropriate?  That way we get `UL` only if it is needed. 

I assume that we want `UC`, `UI`, and `ULL` regardless of what extra wrapping the user does or does not request.  If so, is the same true for `US`?

Reply:
We do not want `WRAP_ITK_SCALAR` here -- only the types that were found to be used / wrapped in the toolkit.

Reply:
Both good points -- the index type list should be `UC;UI;IT` to match the current wrappings for `VectorContainer` as closely as possible. Updating momentarily.